schema.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE medical_histories (id INT PRIMARY KEY, admitted_at TIMESTAMP, status VARCHAR(200), patient_id INT);
+
+CREATE TABLE patients (id INT PRIMARY KEY, name VARCHAR(100), date_of_birth DATE);
+
+CREATE TABLE invoices (id int PRIMARY KEY, total_amount DECIMAL, generated_at TIMESTAMP, payed_at TIMESTAMP, medical_histories_id INT);
+
+CREATE TABLE invoice_items (id INT PRIMARY KEY, unit_price DECIMAL, quantity INT, total_price DECIMAL, invoices_id INT, treatments_id INT);
+
+CREATE TABLE treatments (id INT PRIMARY KEY, type VARCHAR(100), name VARCHAR(100));
```

Comment:
- [ ] Good job implementing this project, however the new tables should be created in a file called `schema_based_on_diagram.sql` file. Kindly rename this file to this name as per the requirement.